api/src/index.ts

```diff
@@ -3,7 +3,7 @@ import { cors } from 'hono/cors'
 import { zValidator } from '@hono/zod-validator'
 import { HTTPException } from 'hono/http-exception'
 import { ZodError } from 'zod'
-import { ConfigStorageService } from './utils/config-storage.js'
+import { ConfigStorageService } from '@shared/config-storage-service'
```

Comment:
TODO: add build process to API so it includes shared packages, and not just reference them. Otherwise, deployment fails.